massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -474,7 +490,22 @@ impl ProtocolWorker {
                 );
             }
             ProtocolCommand::PropagateOperations(storage) => {
-                self.propagate_ops(&storage).await;
+                // Note: should we claim refs locally?
```

Comment:
Wondering about this.

Reply:
That's an important question.

For now I think we can just ignore it because we will end up sending only the ops that are kept in the pool.
In the future (eg. when the pool gets separated out) we should somehow keep the refs to those ops for the duration of the propagation.